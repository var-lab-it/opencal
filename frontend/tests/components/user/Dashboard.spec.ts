import {describe, expect, test} from "vitest";
import {render} from "vitest-browser-vue";
import Dashboard from "../../../src/components/user/Dashboard.vue";

describe('dashboard', () => {
    test('renders the dashboard component', async () => {
        const result = render(Dashboard);

        expect(result).toMatchSnapshot();
    });
});
