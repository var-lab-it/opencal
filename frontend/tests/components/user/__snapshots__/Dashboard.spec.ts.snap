// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`dashboard > renders the dashboard component 1`] = `
{
  "baseElement": <body>
    
  


    <div>
       User Dashboard 
    </div>
  </body>,
  "container": <div>
     User Dashboard 
  </div>,
  "debug": [Function],
  "emitted": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholder": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`logo > renders the logo with url 1`] = `
{
  "baseElement": <body>
    
  


    <div>
      <div
        class="text-center pt-4 pb-5"
      >
        <img
          alt="Logo"
          src="https://example.com/logo.png"
          width="200px"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="text-center pt-4 pb-5"
    >
      <img
        alt="Logo"
        src="https://example.com/logo.png"
        width="200px"
      />
    </div>
  </div>,
  "debug": [Function],
  "emitted": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholder": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
